### PR TITLE
feat(header):create header

### PR DESCRIPTION
**Pull request by beommaker**
https://app.asana.com/0/487815437712560/487815437712584

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "react-dom": "^16.1.1",
+    "react-scripts": "1.0.17"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  }
+}
```
Comment (LEEhajeong) [inline package.json:16]: 👍

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
```
Comment (s2unga9) [inline .gitignore:9]: 👍

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
+  <body>
+    <noscript>
+      You need to enable JavaScript to run this app.
+    </noscript>
+    <div id="root"></div>
+    <!--
```
Comment (LEEhajeong) [inline public/index.html:29]: 💯

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (s2unga9) [inline .gitignore:21]: 👍

**Review by LEEhajeong — APPROVED** [review]
coucou

**Review by s2unga9 — APPROVED** [review]
잘했어요

**Pull request merged**